### PR TITLE
Update compress.bash

### PR DESCRIPTION
**Pull request by ghenrique737**

### CODE REVIEW

File: compress.bash [hunk 1]
```diff
@@ -11,19 +11,19 @@
...
 
-name=ColorOS
+name=AOSP-RRO
 
 
 # Manifest link. https:// is mandatory.
 # Example: https://github.com/cyanogenmod/android
 
-manifest=https://github.com/Qiangong2/patchrom/default.xml
+manifest=https://github.com/AOSP-RRO/manifest -b nougat
```
Comment (msfjarvis) [inline compress.bash:20]: YOU DON'T SPECIFY A BRANCH HERE

Comment (ghenrique737) [reply compress.bash:20]: the branch is nougat, -b nougat is the branch.
https://github.com/AOSP-RRO/manifest

File: compress.bash [hunk 2]
```diff
@@ -11,19 +11,19 @@
...
 
 
 # Manifest branch.
 # Example: cm-14.0
 
-branch=jellybean42
+branch=android-7.0.0_r6
```
Comment (msfjarvis) [inline compress.bash:26]: Branch name belongs HERE, the `-b nougat` there is pure bullshit

**Review by msfjarvis — CHANGES_REQUESTED** [review]

File: compress.bash (hunk 2 above)
Comment (msfjarvis) [reply compress.bash:26]: I don't see such a branch name on the manifest. Have you even bothered to check?

**Pull request closed without merging**